<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <widget class="QWidget" name="page_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>687</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Schedule</string>
      </attribute>
      <layout class="QGridLayout" name="scheduleLayout">
      </layout>
      </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>687</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Networks</string>
      </attribute>
      <layout class="QGridLayout" name="networksLayout">
      </layout>
      </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>687</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout" name="logLayout">
      </layout>
      </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>687</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Shell Commands</string>
      </attribute>
      <layout class="QGridLayout" name="shellCommandsLayout">
      </layout>
      </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
